charts/vald/templates/gateway/backup/ing.yaml

```
@@ -15,7 +15,11 @@
 #
 {{- $gateway := .Values.gateway.backup -}}
 {{- if and $gateway.enabled $gateway.ingress.enabled }}
+{{- if $gateway.ingress.usev1beta1 }}
+apiVersion: networking.k8s.io/v1beta1
```

which one do you think is better?
```suggestion
apiVersion: {{ $gateway.ingress.apiVersion }}
```
and use set the string `networking.k8s.io/v1beta1` to the values.yaml